int16 leftMillimetersPerSecond
int16 rightMillimetersPerSecond